crates/bin/pcli/src/command/tx.rs

```diff
@@ -218,15 +215,69 @@ pub enum TxCmd {
         /// invalid if not already relayed.
         #[clap(long, default_value = "0", display_order = 150)]
         timeout_timestamp: u64,
-
         /// Only withdraw funds from the specified wallet id within Penumbra.
         #[clap(long, default_value = "0", display_order = 200)]
         source: u32,
-
         /// The selected fee tier to multiply the fee amount by.
         #[clap(short, long, value_enum, default_value_t)]
         fee_tier: FeeTier,
     },
+    /// Schedule a Dutch auction, a tool to help accomplish price discovery.
+    #[clap(display_order = 600)]
+    DutchAuctionSchedule {
```

Comment:
Could we refactor the `DutchAuction*` variants into an `AuctionCmd` enum so that we can have these commands:

```rust
pcli auction dutch schedule ...
```

rather than:

`pcli dutch-auction-schedule`

🙏

Reply:
I'll make the change. I originally did that, but then reverted since I noticed the issue description used `pcli tx auction`

Reply:
sorry I missed the tx in the command but we definitely want that

Reply:
the ask here is folding the auction commands into an AuctionCmd enum

Reply:
so just to clarify: pcli tx auction dutch ...

Reply:
yeah noted, was just clarifying!

Reply:
@erwanor there seems to be something wrong with the clap ordering, can you check it out?

Reply:
@TalDerei i'm afk, but can take a look later, what's the problem exactly?

Reply:
when we run `pcli transaction auction dutch...`, the commands don't display the proper flags

Reply:
I figured it out, using clap's `default_value = "0"` prevents the flag from being displayed in the CLI.